build.gradle

```diff
@@ -357,7 +360,7 @@ dependencies {
 
     testCompile group: 'com.h2database', name: 'h2'
     testCompile "com.github.tomakehurst:wiremock:2.19.0"
-    testCompile group: 'org.mockito', name: 'mockito-core', version: '3.6.28'
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '3.4.6'
```

Comment:
Downgraded as the latest jar is causing issues with test cases. The other RD projects have  'testCompile group: 'org.mockito', name: 'mockito-core', version: '3.4.6' jar.

Reply:
Thanks for informing @nitishsingla1234 :)